src/styles/actor/character/_abc-picker.scss

```diff
@@ -0,0 +1,8 @@
+.abc-picker .window-content {
+    flex-flow: column nowrap;
+    height: 100%;
+    padding: 0;
+    > * {
+        display: flex;
+    }
+}
```

Comment:
Maybe, but I'm not sure how much of this is widely applicable. In this case it's because appv2 is packed to the brim with whitespace (padding, gap), which happens to not be appropriate for this particular application.

Reply:
I can't imagine we'd want that whitespace to be predefined on any of our document sheets, tho I suppose that does complicate matters for non document sheets.